frontend/src/app/modules/interventions/map/map.component.html

```diff
@@ -6,14 +6,16 @@
     [longitude]="interventions[0].geoLng"
     (tilesLoaded)="onTilesLoaded()"
   >
-    <agm-marker
-      *ngFor="let intervention of interventions.slice(0, 200)"
-      [latitude]="intervention.geoLat"
-      [longitude]="intervention.geoLng"
-    >
-      <agm-info-window [disableAutoPan]="false">
-        <app-intervention-details [intervention]="intervention" embedded="true"> </app-intervention-details>
-      </agm-info-window>
-    </agm-marker>
+    <agm-marker-cluster imagePath="/assets/m">
+      <agm-marker
+        *ngFor="let intervention of interventions.slice(0, 200)"
```

Comment:
Not the goal of this PR, but I am not sure if we still need `interventions.slice(0, 200)`. This was added when we didn't have clustering and we had 1000 records of FAKE_DATA provided to the map at once, so the map got blocked for a long time before loading ;) Could we maybe clean this up?

Reply:
Will do!

Reply:
OK, done